README.rst

```diff
@@ -12,14 +12,12 @@ torchtext
 
 This repository consists of:
 
-* `torchtext.data <#data>`_: Generic data loaders, abstractions, and iterators for text (including vocabulary and word vectors)
-* `torchtext.datasets <#datasets>`_: Pre-built loaders for common NLP datasets
+* `torchtext.datasets <#datasets>`_: The raw text iterators for common NLP datasets
+* `torchtext.data <#data>`_: Some basic NLP building blocks (tokenizers, metrics, functionals etc.)
+* `torchtext.nn <#nn>`_: NLP related modules
+* `examples <https://github.com/pytorch/text/tree/master/examples>`_: Example NLP workflows with PyTorch and torchtext library.
 
-Note: we are currently re-designing the torchtext library to make it more compatible with pytorch (e.g. ``torch.utils.data``). Several datasets have been written with the new abstractions in `torchtext.experimental <https://github.com/pytorch/text/tree/master/torchtext/experimental>`_ folder. We also created an issue to discuss the new abstraction, and users are welcome to leave feedback `link <https://github.com/pytorch/text/issues/664>`_. These prototype building blocks and datasets in the experimental folder are available in the nightly release only. The nightly packages are accessible via Pip and Conda for Windows, Mac, and Linux. For example, Linux users can install the nightly wheels with the following command::
-
-    pip install --pre torch torchtext -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html  
-
-For more detailed instructions, please refer to `Install PyTorch <https://pytorch.org/get-started/locally/>`_. It should be noted that the new building blocks are still under development, and the APIs have not been solidified.
+Note: the legacy code discussed in `torchtext v0.7.0 release note <https://github.com/pytorch/text/releases/tag/v0.7.0-rc3>`_ has been retired to `torchtext.legacy <#legacy>`_ folder. Those legacy code will not be maintained by the development team, and we plan to fully remove them in the future release. See the Legacy session for more details.
 
 Installation
```

Comment:
In line below, I think Anaconda not the package manager but is more like a distribution whereas conda is the underlying package manager. Should we rephrase it to something like: "We recommend conda as package manager that comes with Anaconda or miniconda"?

Reply:
It's very general statement. I think we can "We recommend Anaconda to access packages"

Reply:
Sure, we can probably leave it as such then to keep the consistency with PyTorch as this is a minor technical detail that users won't mind :)

@cpuhrsch  Just FYI